bofire/strategies/doe/design.py

```diff
@@ -0,0 +1,255 @@
+import warnings
+from copy import deepcopy
+from typing import Callable, Dict, Optional, Union
+
+import numpy as np
+import pandas as pd
+from formulaic import Formula
+from scipy.optimize._minimize import standardize_constraints
+
+from bofire.domain import Domain
+from bofire.domain.constraints import NChooseKConstraint
+from bofire.samplers import PolytopeSampler, apply_nchoosek
+from bofire.strategies.doe.jacobian import JacobianForLogdet
+from bofire.strategies.doe.utils import (
+    constraints_as_scipy_constraints,
+    get_formula_from_string,
+    metrics,
+    nchoosek_constraints_as_bounds,
+)
+
+
+def _domain_for_sampling(domain: Domain) -> Domain:
```

Comment:
I think we can remove this now, as the NChooseK sampler is now able to sample also from domains with NChooseK constraints.